app/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt

```diff
@@ -50,6 +50,7 @@ class SensorReceiver : SensorReceiverBase() {
             PowerSensorManager(),
             PressureSensorManager(),
             ProximitySensorManager(),
+            QuestSensorManager(),
```

Comment:
We need to also add the intent action string and sensor ID to the `skippableActions` below

Reply:
Should be fixed